src/lua/utils.c

```diff
@@ -382,6 +382,12 @@ luaL_setcdatagc(struct lua_State *L, int idx)
 	lua_pop(L, 1);
 }
 
+int
+luaL_getgctotal(struct lua_State *L)
+{
+	return (lua_getgccount(L) << 10) + lua_gc(L, LUA_GCCOUNTB, 0);
```

Comment:
Minor: Maybe it is better to use `* 1024ULL` here, since the `lua_gc()` returns an `int` result that may overflow (theoretically for the GC64 build and 2 TB of Lua memory :)).
Also, I suppose that function should have the `size_t` returned value type, since it has the corresponding semantics.